masterlist.yaml

```diff
@@ -18677,7 +18677,7 @@ plugins:
   - name: 'DawnguardArsenal.esp'
     msg:
       - <<: *patch3rdParty_KPH_ACE
-        condition: 'active("WACCF_Armor and Clothing Extension.esp") and not active("DawnguardArsenal_WACCF-ACE_Patch.esp")'
+        condition: 'active("WACCF_Armor and Clothing Extension.esp") and not active("DawnguardArsenal_ACE_Patch.esp")'
```

Comment:
`DawnguardArsenal_ACE_Patch.esp` is now written with lowercase letters, so `dawnguardarsenal_ace_patch.esp` - see [kryptopyr's Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/19518/). Please change the name of the plugin in this condition accordingly.